src/components/AddFundsInterstitial.js

```diff
@@ -273,7 +271,7 @@ const AddFundsInterstitial = ({ network }) => {
             faucet
           </Title>
           <Row marginTop={30}>
-            <InterstitialButton onPress={() => onAddFromFaucet(network)}>
+            <InterstitialButton onPress={onAddFromFaucet(accountAddress)}>
```

Comment:
ah now that it has an argument again we will need to do the `() => onAddFromFaucuet(accountAddress)` - can pull it out as a const function so it isn't defined inline